dask/imperative.py

```diff
@@ -183,6 +183,25 @@ def get(dsk, keys, get=None, **kwargs):
     return get(dsk2, keys, **kwargs)
 
 
+def compute(*args, **kwargs):
+    """Evaluate several ``Value``s at once.
+
+    Examples
+    --------
+    >>> a = value(1)
+    >>> b = a + 2
+    >>> c = a + 3
+    >>> compute(b, c)  # Compute both simultaneously
+    (3, 4)
+    """
+    if len(args) == 1:
```

Comment:
I don't like this sort of inference -- better to always return a tuple. The `compute` method can always be used if you want a single value, and this makes it hard to use programmatically.